tests/pcap-log-lz4-01/test.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+  features:
+    - liblz4
+  min-version: 8
+      
+pcap: ../bug-2482-01/proxyCONNECT_443.pcap
+
+args:
+  - --runmode=single
+
+checks:
+    - shell:
+        args: ls -l log.pcap.15233891*.lz4 | wc -l | xargs
```

Comment:
Why do we need xargs ?

Reply:
What I've seen it do is strip leading whitespaces `           1` -> `1`. Don't know if it is needed.

Reply:
`-l` should ideally count lines I think so as long as the spaces leading or trailing are on the same line, it should work.